src/parse.rs

```diff
@@ -163,33 +171,55 @@ struct CompilerContext<'pt> {
 impl<'pt> CompilerContext<'pt> {
     fn new(grm: &'pt YaccGrammar, input: &'pt str) -> CompilerContext<'pt> {
         CompilerContext {
-            classes: HashMap::new(),
             symbols: HashMap::new(),
+            bytecode: Vec::new(),
+            labels: HashMap::new(),
             grm:     grm,
             input:   input,
             cur_cls: "global".to_string(),
             cur_fn:  "global".to_string(),
         }
     }
 
+    // The bytecode length of the current class.
+    // Useful for building jump labels
+    fn get_bytecode_length(&self) -> usize {
```

Comment:
The Rust convention would be `bytecode_len` I think. Although given it's not a public function, I'm not really sure it's worth having as a separate function? `self.bytecode.len()` is shorter than `self.get_bytecode_length()` ;)